src/Test/Unit/Config/Validator/Deploy/DatabaseSplitConnectionTest.php

```diff
@@ -78,14 +79,19 @@ public function testMessageValidate()
             ->willReturn($dbConfiguration);
         $this->resultFactoryMock->expects($this->once())
             ->method('error')
-            ->with('Split database configuration was detected in the property DATABASE_CONFIGURATION of the'
-                . ' file .magento.env.yaml:' . PHP_EOL
-                . '- connection: checkout' . PHP_EOL
-                . '- connection: sales' . PHP_EOL
-                . '- slave_connection: checkout' . PHP_EOL
-                . '- slave_connection: sales' . PHP_EOL
-                . 'Magento Cloud does not support a custom split database configuration,'
-                . ' such configurations will be ignored');
+            ->with(
+                'Detected split database configuration in the DATABASE_CONFIGURATION property of the'
+                    . ' file .magento.env.yaml:' . PHP_EOL
+                    . '- connection: checkout' . PHP_EOL
+                    . '- connection: sales' . PHP_EOL
+                    . '- slave_connection: checkout' . PHP_EOL
+                    . '- slave_connection: sales' . PHP_EOL
+                    . 'Magento Cloud does not support a custom split database configuration,'
+                    . ' such configurations will be ignored',
+                'Remove custom connections for split databases from the DATABASE_CONFIGURATION variable '
```

Comment:
```suggestion
                'Update the DATABASE_CONFIGURATION variable in the `.magento.env.yaml` file to remove custom connections for split databases.'
```

Aren't the only valid connections "checkout" and "sales".   Should the error message be more specific.  Magento Cloud supports split database connections for `checkout` and `sales`  databases only?